src/main/java/uk/gov/justice/laa/crime/cfecrime/utils/FullMeansTestOutcomeCalculator.java

```diff
@@ -10,48 +10,81 @@
 @Slf4j
 public class FullMeansTestOutcomeCalculator {
 
-    private FullMeansTestOutcomeCalculator(){
-    }
-
     public static MeansTestOutcome getFullMeansTestOutcome(FullAssessmentResult result, CaseType caseType, MagCourtOutcome magCourtOutcome){
-        log.debug("Get the outcome of the Full Means Test. Inputs: result = {} caseType = {} magCourtOutcome = {}", result, caseType, magCourtOutcome);
-
-        //Fail result: Heard In Magistrates Court
-        Set<CaseType> caseTypesHeardInMagistratesCourt = Set.<CaseType>of(CaseType.COMMITAL, CaseType.SUMMARY_ONLY, CaseType.EITHER_WAY);
+        log.debug("FullMeansTestOutcome start. Inputs: caseType = {} magCourtOutcome = {} result = {}", caseType, magCourtOutcome, result);
 
         MeansTestOutcome meansTestOutcome = null;
-        if (result != null && caseType != null && magCourtOutcome != null) {
 
-            if (result == FullAssessmentResult.PASS) {
-                    //All Eligible with no contribution
+        if (result == null) {
```

Comment:
Yes, a nice idea to represent the table as data. It could go further and be a hash table with the key being the inputs - then you'd avoid all if/switch branching! I'd definitely give it a try if we didn't already have a reasonable solution ready to merge, but the approach is one to consider on the next one like this.